### ISSUE TITLE
Memory game set of modules: Must be able to paste list words into the columns

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#1552]
I know that the original design was to paste into the list, and this was discarded when it was hard to match the column entries. But user testing has shown that teachers need to be able to paste a list of words rather than going into the individual cells.

Please find a way to paste words into the lists.

**Comment by MendyBerger** [comment]
@dakom do you happen to remember the challenges you had when trying to implement this in the past?

**Comment by dakom** [comment]
I don't think this particular issue was a problem from my side, just wasn't done... on one foot, should be possible to detect newlines and then split it out over the remaining cells.

In some modes like "Riddle" it probably shouldn't happen automatically.

**Comment by mastapegs** [comment]
Hey there @MendyBerger, would you like for me to take care of this issue for you?

I'd use an event handler to handle the paste event in those columns and write up some logic to split the clipboard's content by newlines, and render that data into the columns.

I think this is something I may be able to take care for you if you'd like :-)

![image](https://user-images.githubusercontent.com/13839150/147717766-62b1981c-1689-4208-819d-3d23206be496.png)

**Comment by corinnewo** [comment]
This issue has become really important, with many teachers finding it cumbersome to add words one at a time, especially when they have ready-made lists.
